actions/mysql_show_table_status_action_test.go

```diff
@@ -82,6 +82,19 @@ func TestShowTableStatus(t *testing.T) {
 				nil, "", "",
 			}, actual[1])
 
+		case "10.2":
```

Comment:
this case can be added to the case below.

Reply:
They are different.